### PR TITLE
Add Celeste port

### PR DESCRIPTION
**Pull request by JohnnyonFlame**
# New Port for Celeste

## Game Information
- **Title**: Celeste
- **URL**: https://github.com/JohnnyonFlame/FNAPatches/blob/master/Celeste/CelestePatches.cs

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [x] AmberELEC
- [x] ArkOS
- [x] ROCKNIX
- [x] muOS

### Resolution Tests
Test all major resolutions:
- [x] 480x320
- [x] 640x480
- [ ] 720x720 (RGB30)
- [x] Higher resolutions (e.g., 1280x720)

### CODE REVIEW

File: ports/celeste/Celeste.sh [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+source $controlfolder/control.txt
+source $controlfolder/tasksetter
+
+get_controls
+
+export gamedir="/$directory/ports/celeste"
+export gameassembly="Celeste.exe"
+cd "$gamedir/gamedata"
+
```
Comment (Cebion) [inline ports/celeste/Celeste.sh:23]:
```suggestion
> "$gamedir/log.txt" && exec > >(tee "$gamedir/log.txt") 2>&1
```

File: ports/celeste/Celeste.sh [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+		--title "First Time Setup" \
+		../repack.src
+	if [[ $? != 0 ]]; then
+		exit
+	fi
+fi
+
+# first_time_setup
+$GPTOKEYB "mono" &
+$TASKSET mono Celeste.exe 2>&1 | tee "$gamedir/log.txt"
```
Comment (Cebion) [inline ports/celeste/Celeste.sh:69]:
```suggestion
$TASKSET mono Celeste.exe"
```

Comment (JohnnyonFlame) [reply ports/celeste/Celeste.sh:69]: Typo here, trailing "

[the remark this answers is not in the document]
Comment (JohnnyonFlame) [reply ports/celeste/Celeste.sh:69]: > Would it be possible to use `> "$GAMEDIR/log.txt" && exec > >(tee "$GAMEDIR/log.txt") 2>&1` to log the execution of the script into log.txt, as it is suggested in the launch script template ?

I want certain parts of the script to be logged differently, the rest of the script (not logged) is noise mostly.

**Comment by Cebion** [comment]
> I don't want the entire script to be logged the same, I want to keep certain logs separate for debugging reasons, are those changes strictly necessary?

If there is a reason behind it, we can of course get behind it. Else we try to set a standard =)

**Pull request merged**